antora-playbook.yml

```diff
@@ -39,4 +39,4 @@ ui:
   bundle:
     url: https://github.com/camptocamp/devops-stack-antora-ui/releases/download/master-LATEST/ui-bundle.zip
     snapshot: true
-  supplemental_files: ./supplemental_ui
+  supplemental_files: ./docs/supplemental_ui
```

Comment:
Why did this change?

Reply:
Since the supplemental UI folder is only used by Antora to generate the documentation, I think it’s cleaner to put it in the docs directory.